apps/prairielearn/python/prairielearn.py

```diff
@@ -733,7 +724,7 @@ def get_color_attrib(element, name, *args):
             )
 
 
-def numpy_to_matlab(A, ndigits=2, wtype="f"):
+def numpy_to_matlab(np_object, ndigits=2, wtype="f"):
```

Comment:
I hate to say this, but this is a change to this function's signature and technically breaking (since the arguments are not positional-only). I.e. `numpy_to_matlab(A=my_matrix)` will now throw an error with this change. Though I think this will affect a very small percentage of people.

Reply:
I just ran a quick grep over things. There are very few callers of this function and none of them are using named arguments. I think this change is safe to make.